api/agentpb/agent.proto

```diff
@@ -58,6 +58,10 @@ message SetStateRequest {
     repeated string env = 5;
     map<string, string> text_files = 6;
     repeated string redact_words = 7;
+    // TLS in enabled if true.
```

Comment:
these fields actually is not required for processes

Reply:
Yes, removed. Thanks.